content/en/docs/tasks/configure-pod-container/resize-container-resources.md

```diff
@@ -0,0 +1,261 @@
+---
+title: Resize CPU and Memory Resources assigned to Containers
+content_type: task
+weight: 30
+min-kubernetes-server-version: 1.27
+---
+
+
+<!-- overview -->
+
+{{< feature-state state="alpha" for_k8s_version="v1.27" >}}
+
+This page assumes that you are familiar with [Quality of Service](/docs/tasks/configure-pod-container/quality-service-pod/)
+for Kubernetes Pods.
+
+This page shows how to resize CPU and memory resources assigned to containers
+of a running pod without restarting the pod or its containers. A Kubernetes node
+allocates resources for a pod based on its **requests**, and limits the pod's
+resource usage based on the **limits** specified in the pod's containers.
+
+The following changes enable in-place resize of container resources:
+- Container's resource **requests** and **limits** are now **mutable** for CPU
+and memory resources.
```

Comment:
Not a strong opinion, but I'd like to format bulletin items by having the following lines indented. For example:

```suggestion
- Container's resource **requests** and **limits** are now **mutable** for CPU
  and memory resources.
```

Reply:
fixed